__main__.py

```diff
@@ -1,10 +1,21 @@
 import asyncio
+import logging
 import os
 import sys
 
 import hangups
 
 
+logger = logging.getLogger('hangbot')
+logger.setLevel(logging.DEBUG)  # this could depend on the environment
+
+stdout_handler = logging.StreamHandler()
+stdout_handler.setLevel(logging.DEBUG)
+stdout_handler.setFormatter(
+    logging.Formatter('%(asctime)s - %(name)s - [%(levelname)s] %(message)s'))
```

Comment:
This is pretty long and tends to wrap in an 80 character terminal window. Could maybe remove ms from time or replace a dash with a colon to make it a little shorter.